app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
```suggestion
  devise :database_authenticatable, :registerable, :validatable
```
Giới hạn lại các tính năng của pull
Chỗ này e có thể tìm hiểu mỗi tính năng hỗ trợ các hàm nào, validate nào trong model